tests/fn_boundary.json

```diff
@@ -291,7 +357,7 @@
       "expect": [
         {
           "id": "o1",
-          "decimal": 0.95
```

Comment:
If input value is 1.0 then how is it possible to get 0,95? I think it is an error in the test values(same for highBoundary).